core/lib/zksync_core/src/lib.rs

```diff
@@ -723,38 +805,86 @@ pub async fn initialize_components(
             .default_priority_fee_per_gas;
         let l1_chain_id = genesis_config.l1_chain_id;
 
-        let eth_client = PKSigningClient::new_raw(
-            operator_private_key.clone(),
-            diamond_proxy_addr,
-            default_priority_fee_per_gas,
-            l1_chain_id,
-            query_client.clone(),
-        );
+        let gkms_op_key_name = std::env::var("GOOGLE_KMS_OP_KEY_NAME").ok();
+        let use_gkms_signing_client = gkms_op_key_name.is_some();
 
-        let eth_client_blobs = if let Some(blob_operator) = eth_sender_wallets.blob_operator {
-            let operator_blob_private_key = blob_operator.private_key().clone();
-            let client = Box::new(PKSigningClient::new_raw(
-                operator_blob_private_key,
+        let eth_tx_manager_actor;
+        if use_gkms_signing_client {
```

Comment:
Maybe use `let eth_tx_manager_actor = if use_gkms_signing_client {` here if you want, this way it is more clear what `eth_tx_manager_actor` is.

Reply:
done